telegram/ext/filters.py

```diff
@@ -2545,3 +2546,60 @@ def filter(self, message: Message) -> bool:
 
 VOICE = _Voice("filters.VOICE")
 """Messages that contain :attr:`telegram.Message.voice`."""
+
+
+class Mention(MessageFilter):
+    """Mention messages. If list of username or chat_id or :attr:`telegram.User` objects passed,
+    it filters messages to only allow those whose contains one of username
+    or chat_id or :attr:`telegram.User`.
+    Note:
+        Usernames only allowed with @ prefix. Example: "@username"
+        Chat id only allowed with positive numbers. Example: 123456
+    Examples:
+        A simple use case for passing username is to allow only messages that were sent with
+        mention username:
+        MessageHandler(filters.Mention("@username"), callback_method)
+        MessageHandler(filters.Mention(["@username",123456, `user` ), callback_method)
+    Args:
+        mentions (Union[:obj:`str` ,:obj:`str` ,:attr:`telegram.User` ] ||
+        Collection[Union[:obj:`str` ,:obj`str` ,:attr:`telegram.User` ]]):
+        Which messages to allow with username or chat_id or :attr:`telegram.User` object.
+        If one of them matched it will allow. If not specified,
+        will allow any text message with mention.
```

Comment:
```suggestion
        If one of them matched it will allow.
```

The argument is not optional :)

Reply:
Fixed